app/components/task/details/output/task-outputs.component.ts

```diff
@@ -21,63 +21,84 @@ const outputTabs = [
     selector: "bl-task-outputs",
     templateUrl: "task-outputs.html",
 })
-export class TaskOutputsComponent implements OnChanges {
+export class TaskOutputsComponent implements OnChanges, OnDestroy {
     @Input() public jobId: string;
 
-    @Input() public taskId: string;
+    @Input() public task: Task;
 
     public OutputType = OutputType;
```

Comment:
outputType

Reply:
no this is to pass the enum to the template. I think it is better to keep the same case

Reply:
oh ok cool. if you are happy then i am happy :)